docker-compose.yml

```diff
@@ -61,10 +61,24 @@ services:
     networks:
       - smart-health-links-portal-network
 
+  hapi-fhir:
```

Comment:
We should also add the flag to enable IPS within the instance. This isnt enabled by default. Example can be found here: https://github.com/jembi/platform/blob/main/fhir-datastore-hapi-fhir/docker-compose.yml#L23

Reply:
Yes, Sorry about that, my mistake.